types/node/child_process.d.ts

```diff
@@ -3,6 +3,16 @@ declare module "child_process" {
     import * as net from "net";
     import { Writable, Readable, Stream, Pipe } from "stream";
 
+    type Serializable = string | object | number | boolean;
+    type SendHandle = net.Socket | net.Server;
+
+    interface ChildProcessIPC extends ChildProcess {
```

Comment:
it's possible to create IPC channels also via `spawn()` by specifying e.g `[0, 1, 2, 'ipc']`, therefore `ChildProcessIPC` is not a special case for `fork()` only